docs/src/main/sphinx/security/internal-communication.md

```diff
@@ -4,10 +4,11 @@ The Trino cluster can be configured to use secured communication with internal
 authentication of the nodes in the cluster, and to optionally use added security
 with {ref}`TLS <glossTLS>`.
 
+(internal-secret)=
 ## Configure shared secret
 
 Configure a shared secret to authenticate all communication between nodes of the
-cluster. Use this configuration under the following conditions:
+cluster. You must configured a shared secret under the following conditions:
```

Comment:
Configure

Reply:
Shortened the two sentences into one and fixed